Libplanet.Net/Consensus/ContextTimeoutOption.cs

```diff
@@ -0,0 +1,119 @@
+using System;
+
+namespace Libplanet.Net.Consensus
+{
+    /// <summary>
+    /// A options class to configure <see cref="Context{T}"/> timeout for each <see cref="Step"/>.
+    /// </summary>
+    public class ContextTimeoutOption
+    {
+        private int _proposeSecondBase = 5;
+        private int _preVoteSecondBase = 5;
+        private int _preCommitSecondBase = 5;
+        private int _proposeMultiplier = 1;
+        private int _preVoteMultiplier = 1;
+        private int _preCommitMultiplier = 1;
+
+        public int ProposeSecondBase
+        {
+            get => _proposeSecondBase;
+            set
+            {
+                if (value <= 0)
+                {
+                    throw new ArgumentOutOfRangeException(
+                        nameof(value),
+                        "ProposeSecondBase must be greater than or equal to 0."
+                    );
+                }
+
+                _proposeSecondBase = value;
+            }
```

Comment:
Might be just my personal preference, but I'm not a big fan of setters where mutation doesn't make much sense. 🙄
Is there a context in which we would want to change the option on the fly? If not, I'd prefer using default values for a constructor.

Reply:
Hmm, I also don't know if there's any such use-case, and furthermore, if it should be permitted.

Reply:
Changed to the constructor style.
https://github.com/planetarium/libplanet/commit/9568ed60af44d865baaa9ba196700cdbb5398253#diff-95065cc3ba907122e5352fa2ee987c580d32bfe93d8c8e5a71b70a23e2767aeb